railties/test/generators/migration_generator_test.rb

```diff
@@ -196,6 +196,24 @@ def test_add_migration_with_references_options
     end
   end
 
+  def test_add_migration_with_references_options_when_primary_key_uuid
+    Rails.application.config.generators do |g|
+      g.orm :active_record, primary_key_type: :uuid
```

Comment:
Instead of changing the config pass the option to the `run_generator` method like line 267 is doing. Also move the tests to be closer to the other one that also change the primary key type.